beacon-chain/sync/validate_attester_slashing.go

```diff
@@ -53,7 +57,20 @@ func (s *Service) validateAttesterSlashing(ctx context.Context, pid peer.ID, msg
 	if err := blocks.VerifyAttesterSlashing(ctx, headState, slashing); err != nil {
 		return pubsub.ValidationReject, err
 	}
-
+	slashedVals := blocks.SlashableAttesterIndices(slashing)
```

Comment:
Better to move this up a little and check if `slashedVals == nil` then you can return the nil message error above and avoid the double check for nil